website/content/docs/install/ports.mdx

```diff
@@ -1,9 +1,8 @@
 ---
 layout: docs
-page_title: Required Ports
+page_title: Port Requirements
```

Comment:
I think switch it back since there aren't specific requirements about the ports themselves, just specific numbered ports that are required.

Reply:
If improving the search ability of the content is the goal, then "port requirements" or "port number requirements" might be the better title. People looking for this type of information would most likely search on "requirements". But I see your point.